_data/current.yml

```diff
@@ -82,6 +82,13 @@
   location: Online
   dates: "June 28-29, 2021"
   url: https://waset.org/critical-computer-based-systems-conference-in-june-2021-in-london?utm_source=testingconferences
+  
+- name: Agile, DevOps and Testing: New Directions in Methods and Tools
```

Comment:
The build is failing because you can't use `:` in names. It denotes something specific in jekyll. I'll help by fixing it and then the build should work.